spec/features/review_card_spec.rb

```diff
@@ -14,8 +14,8 @@
 
   scenario 'enter the incorrect translation' do
     sign_in(user)
-    fill_in 'input_text', with: 'Приветик'
+    fill_in 'input_text', with: 'Пюратрет'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.